### PR TITLE
Gallery card update

### PR DESCRIPTION
**Pull request by AndreyGrin**
Hey, added requested styles (issue [#230](https://github.com/kodadot/nft-gallery/issues/230)).
This is currently mock Price and EMOTE. Is it possible to add from back-end next params: `hasPrice`, `price` and number for EMOTE (this is amount of likes or so?). Implemented without  https://vuesax.com/docs/components/Card.html#default. Check plz styles and let me know what do you think. I've added zoom animation for video previews as well to be consistent.

### CODE REVIEW

**Comment by yangwao** [comment]
Siiick! Like it!
I mean just manage to hide the tittle so it could roll over picture once there is hover.
For `hasPrice`, `price` cc @vikiival

**Comment by vikiival** [comment]
@AndreyGrin [NFT has already field price field ](https://github.com/kodadot/nft-gallery/blob/518de4b7a05eba9a37a0450951bbf5224410ffe2/dashboard/src/components/rmrk/service/scheme.ts#L131)

`hasPrice` why ?

> number for EMOTE

**Comment by AndreyGrin** [comment]
@vikiival `hasPrice` used on ntf-item page but actually, just `price` will be enough.
@yangwao changed animation. Do you consider mobile version as well? For mobile, I think, it better to leave NFT title visible.

**Comment by vikiival** [comment]
> For mobile, I think, it better to leave NFT title visible.

Yeah It wold be perfect to leave it visible (you have no hover on mobile)

**Comment by vikiival** [comment]
@AndreyGrin I am not sure if it's because of this pr but [detail seems a bit broken]( https://deploy-preview-289--nftkodadot.netlify.app/rmrk/detail/7005825-A46C0A25BE37522E15-BOTS-DAY_OLD-BANANA_PUDDING-0000000000000001)

**Comment by AndreyGrin** [comment]
@vikiival last commit should prevent any impact on other pages

**Comment by AndreyGrin** [comment]
Guys, if you confirm design implementation, I will update code to support `price` and `emoteCount` params but it will make labels disappear until yours BE will add it's part. Please confirm @yangwao

**Comment by yangwao** [comment]
Hey checking in. 

Seems the search is gone? 

image

Meanwhile please fetch recent `main` into your PR to avoid some conflicts in the merge.

**Comment by AndreyGrin** [comment]
`main` was up to date. Fixed styles and added a bit cleaner animation for sliding-up not title.

**Comment by yangwao** [comment]
Hey, can you re-check one more time? There was recent merge to the  `main`, where you should see sort-filter in search https://github.com/kodadot/nft-gallery/commit/d7eaa8057cfbcc0f85871c7e6ada84a345eab345

**Comment by yangwao** [comment]
oh found something weird semi-state.
https://user-images.githubusercontent.com/5887929/114833250-a8c1db00-9dcf-11eb-9f24-37375aacf8a0.mov

**Comment by yangwao** [comment]
> @AndreyGrin [NFT has already field price field ](https://github.com/kodadot/nft-gallery/blob/518de4b7a05eba9a37a0450951bbf5224410ffe2/dashboard/src/components/rmrk/service/scheme.ts#L131)
> 
> `hasPrice` why ?
> 
> > number for EMOTE
> > due to current architecture it's a bit complicated but we are working on migration
> 
> However, if you want to be make the consistent put there something like `emoteCount` field I will update it later

Hey by this price should be there? 
@vikiival will add `emoteCount`, so you can prepare it to be ready :)

**Comment by AndreyGrin** [comment]
https://user-images.githubusercontent.com/2453346/114854963-0c5b0100-9dee-11eb-8134-45ebce2e0b38.mov

in order to get rid of the animation glitch but to keep card structure same, applied `overflow:hidden `for the card in the gallery, but had to change animation for labels to show them in full.

**Comment by AndreyGrin** [comment]
Hey, @yangwao! Any news on the PR?

**Comment by vikiival** [comment]
> Hey, @yangwao! Any news on the PR?

Hey @AndreyGrin please make sure that you are using correct prices for NFTs (there is field `price` on `nft`) and also conditionally render the `emoteCount`

**Review by AndreyGrin — COMMENTED** [review]
@vikiival check comments to the code. I believe I use correct not props

File: dashboard/src/components/rmrk/Gallery/Gallery.vue [hunk 1]
```diff
@@ -31,9 +32,10 @@
                     @error="onError"
                   />
                 </figure>
+                <span v-if="nft.price" class="card-image__price">{{nft.price}} KSM</span>
```
[the remark this answers is not in the document]
Comment (vikiival) [reply dashboard/src/components/rmrk/Gallery/Gallery.vue:35]: Use `<Money />` component please

File: dashboard/src/components/rmrk/Gallery/Gallery.vue [hunk 2]
```diff
@@ -22,6 +22,7 @@
           <div class="card nft-card">
             <router-link :to="{ name: 'nftDetail', params: { id: nft.id }}" tag="div" class="nft-card__skeleton">
               <div class="card-image">
+                <span v-if="nft.emoteCount" class="card-image__emotes">{{nft.emoteCount}}</span>
```
[the remark this answers is not in the document]
Comment (vikiival) [reply dashboard/src/components/rmrk/Gallery/Gallery.vue:25]: Awesome !

**Comment by AndreyGrin** [comment]
@vikiival changed Price to use <Money> component

**Comment by yangwao** [comment]
Hey checking @AndreyGrin 
I don't see any price here on this NFT on gallery view 
https://deploy-preview-289--nftkodadot.netlify.app/rmrk/gallery 
image

We would like close this so it can go to our 7k changelog

**Comment by vikiival** [comment]
We didn't project price from query. Pushed the fixed.

@yangwao /@AndreyGrin I would maybe remove USD value. But it's personal prefference

Screenshot 2021-04-20 at 13 16 42

**Comment by yangwao** [comment]
Yeah, checking on rarible.
USD -> $ ? 
image

**Comment by yangwao** [comment]
huh wha's this 🗡️  🥶
image

**Comment by AndreyGrin** [comment]
@yangwao what exactly? Unit?

**Comment by yangwao** [comment]
let's merge it! gonna process payout! 
I mean, if you are interesting and be add value to KodaDot NFT gallery, I'm inviting you to KodaDot:Anteriors (Frontend stuff) channel https://t.me/joinchat/eyLAW64RKsNlZjVk - link expires in day

**Comment by yangwao** [comment]
let's merge it! gonna process payout!
I mean, if you @AndreyGrin  are interested and be add value to the KodaDot NFT gallery, 
I'm inviting you to KodaDot:Anteriors (Frontend stuff) channel https://t.me/joinchat/eyLAW64RKsNlZjVk - link expires in day

**Pull request merged**